### ISSUE TITLE
Miscounting of children for a class or not showing children for a class

### ISSUE THREAD

**Issue by anitacaron** [EBISPOT/ols4#382]
In some cases, a class has a count of children, but expanding the branch is impossible.

![Screenshot 2023-06-05 at 11 48 39](https://github.com/EBISPOT/ols4/assets/2208124/af7d110c-9c08-4f4e-867f-4d1076aca054)

**Comment by jamesamcl** [comment]
Which ontology is this?

**Comment by anitacaron** [comment]
It's in the FOVT https://www.ebi.ac.uk/ols4/ontologies/fovt/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FUBERON_0016887

**Comment by jamesamcl** [comment]
I think the counts are correct because if you click the term you can expand it on the term page. Must be the cypher query not returning the children.

IIRC the cypher query has a limit; maybe it needs to be increased.

**Comment by anitacaron** [comment]
This happens when I click on "Show all siblings".

**Comment by ghost** [comment]
Also noticed this issue- I'm unsure how to interpret the counts in OLS4.

For example, '[lamellar body](http://purl.obolibrary.org/obo/GO_0042599)' in GO:

<img width="422" alt="Screenshot 2023-06-14 at 10 48 57" src="https://github.com/EBISPOT/ols4/assets/95643797/199cb60b-04ac-4e93-9870-aacf6d2f88b7">

- The counts ostensibly represent the count of all subclasses of the classes that appear next to them (both indirect and direct). For example, a user likely interprets the view above as there being 6 classes under 'lamellar body'.
- Some classes do not have the option to expand to view the subclasses in situ, [as noted above](https://github.com/EBISPOT/ols4/issues/382#issue-1741529839). For example, 'alveolar lamellar body' has (1) next to it, but a user has to click on the term to refocus on the term (reloading the page) and only then the subclass of 'alveolar lamellar body' can be viewed. There is no triangle next to it to expand like there is for 'lamellar body membrane'.


EDIT: On closer inspection, the counts are reasonably distinct counts. In this case, the subclasses under 'alveolar lamellar body' and 'epidermal lamellar body' are the same subclasses under 'lamellar body membrane'. This was not immediately apparent since these classes can't be expanded in the present view without reloading (refocusing) the page.

The pending question is why are some classes not "expandable"?

**Comment by jamesamcl** [comment]
Update on this - I figured out how to interpret the counts, at least. They're displaying the number of descendants but _only counting them once_ (i.e. *distinct* descendants). There are six _unique_ descendants of `lamellar body`, because the other two are duplicates (in the hierarchy) of `alveolar lamellar body membrane` and `epidermal lamellar body membrane`.

<img width="634" alt="Screenshot 2023-06-14 at 22 46 11" src="https://github.com/EBISPOT/ols4/assets/454895/346587de-eafc-4edc-b9aa-bb16f2a3e56a">

Why `alveolar lamellar body` and `epidermal lamellar body` aren't expanded by default I am not so sure yet.

EDIT: I just realised @bvarner-ebi also realised they were distinct counts - oh well at least we confirmed each others' hypothesis :D
